Setup mobile version

ademibowale opened the pull request:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

I did this project by
1.  Pulled my previous work from the main branch
2. Modified my html file to fit the desktop version
3. Modified my CSS styling to look exactly like the Figma template
4. Add some images to match the desktop version project
5. Commit my changes to the branch
6.  Push my work to have the latest work on my repo

KayLemba commented on index.html:
```diff
@@ -12,30 +12,25 @@
+            <li><img src="images/vector.png" class="social-logo logon" alt="my logo"></li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
```
It seems like your social media icons are not displaying or there links are broken please kindly fix this issue. 

Your design: 
![Screenshot from 2022-05-23 21-39-30](https://user-images.githubusercontent.com/79506105/169877560-c12b84f6-a02e-4445-ba32-364f54142da5.png)
Please fix this.

KayLemba commented on README.md:
```diff
@@ -1,42 +1,30 @@
-
-## 🤝 Contributing
-
+# portfolio
+
+Project Name
+my-portfolio
+MOBILE-VERSION
+
+Built With
+Major languages HTML and CSS
+Frameworks
+Stylelint e.t.c
+Authors
+👤 Author1
+
+GitHub: @ademibowale
+Twitter: @ademibowale1
+LinkedIn: https://www.linkedin.com/in/adebowale-adegboye-143568221/
+🤝 Contributing
```
It would be nice if you added a description about the project you are building and also add how another person can access your repository locally please follow this link: https://github.com/microverseinc/readme-template

ademibowale replied:
please my social media images is show very clearly 
![Screenshot (12)](https://user-images.githubusercontent.com/92458236/169895006-c470ee75-2b07-4287-be5c-dd19eb34dc8f.png)

KayLemba reviewed (changes requested):
Hi @ademibowale  👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 - Please add Proffessional and descriptive PULL REQUEST. Always start your Pull request phases with a capital letter,
- Please add a screenshot showcasing your UI  to read README file to make it look more proffessional
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

ademibowale commented:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changesI did this project by


I did this project by
1 Pulled my previous work from the main branch
2. Modified my html file to fit the mobile version
3. Modified my CSS styling to look exactly like the Figma template
4. Add some images to match the mobile version project
5. Commit my changes to the branch
6. Push my work to have the latest work on my repo

youmari commented on README.md:
```diff
@@ -1,42 +1,31 @@
+MOBILE-VERSION
 
-# Project Name
+ Project Description.
```
- [x] as the previous reviewer asked, it would be better if you add a description about your projects for example `This is my portfolio website where I show my recent projects and my contact information built with HTML and CSS ` so please try to do that

sja-thedude replied:
Implemented 👍

youmari commented on README.md:
```diff
@@ -1,42 +1,31 @@
+Stylelint e.t.c
 
+👤 Author1
+ADEBOWALE ADEGBOYE
+GitHub: @ademibowale
+Twitter: @ademibowale1
+LinkedIn: https://www.linkedin.com/in/adebowale-adegboye-143568221/
```
- [x] please try to fix your social media information by using the markdown language provided in the readme template 
example : `[put the name of the social media like : Github](put the URL of your profile here )` and it will be display as clickable link

youmari commented on README.md:
```diff
@@ -1,42 +1,31 @@
-
-👤 *Author1*
-
-- GitHub: [@adebowale](https://github.com/adebowale)
-- Twitter: [@adebowale](https://twitter.com/adebowale)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/adebowale)
-
```
- [x] please add these social media accounts as the previous reviewer didn't require to delete them

youmari commented on README.md:
```diff
@@ -1,42 +1,31 @@
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
- [x] I see that you are using the MIT file so it would be better if you keep the `License` section

sja-thedude replied:
Implemented 👍

youmari commented on index.html:
```diff
@@ -12,30 +12,25 @@
-            <li><img src="images/vector.png8.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png4.png " alt="my logo " class="social-logo"></li>
-            <li><img src="images/vector.png2.png " alt="my logo " class="social-logo"></li>
+            <li><img src="images/vector.png3.png" class="social-logo" alt="my logo"></li>
+            <li><img src="images/vector.png" class="social-logo logon" alt="my logo"></li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
```
- [ ] I can't see your social media icons as the previous reviewer asked  please try to fix this issue 

![image](https://user-images.githubusercontent.com/61790587/169910600-c4379234-f79f-4ff8-8d48-8ee2615f1be8.png)

sja-thedude replied:
Implemented 👍

youmari reviewed (changes requested):
Hi @ademibowale , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No liner error 
✔️ Great title and short summary 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ademibowale commented:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes


I did this project by

- [ ] Pulled my previous work from the main branch
- [ ] Modified my html file to fit the desktop version
- [ ] Modified my CSS styling to look exactly like the Figma template
- [ ] Add some images to match the desktop version project
- [ ] Commit my changes to the branch
- [ ]Push my work to have the latest work on my repo

ademibowale commented:
My code follows the style guidelines of this project
 I have performed a self-review of my code
 I have commented my code, particularly in hard-to-understand areas
 I have made corresponding changes to the documentation
 My changes generate no new warnings
 I have added tests that prove my fix is effective or that my feature works
 New and existing unit tests pass locally with my changes

ademibowale commented:
- My code follows the style guidelines of this project
- I have performed a self-review of my code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have tested manually to prove my fix is effective and that my feature works

- Modified my html file to fit the mobile version
- Modified my CSS styling to look exactly like the Figma template
- Add some images to match the mobile version project
- Commit my changes to the branch
- Push my work to have the latest work on my repo

sja-thedude reviewed (approved):
## Project Approved 👍 

Hi @ademibowale, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![giphy](https://user-images.githubusercontent.com/70269507/159485323-11dfc0c5-bd53-4956-b55a-2fc42f7a3845.gif)

Cheers and Happy coding!👏👏👏

## Optional: It would be better to write your contact information in readme this way:
Example : 
Github: [ademibowale](https://github.com/ademibowale) 
Twitter: [ademibowale](https://twitter.com/Ademibowale1) 
Linkedin: [ademibowale](https://www.linkedin.com/in/adebowale-adegboye-143568221/) 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
